100Change2017/etl/compose-and-upload

```diff
@@ -41,8 +41,10 @@ Command-line options:
                                   the torque wiki.  We don't automatically do that because we want to
                                   overwrite the configuration out there.
 
-  --pare N                        Reduce the number of items to 1/N rows, for quick
-                                  testing and development
+  --pare ARG                      If ARG is a number, reduce the number of items to 1/ARG.  If
```

Comment:
It might be good to be explicit about things: if you use "+" or "@" to specify which rows to include, then no *other* keys are included.  I.e., it's not just "include", it's "include only".